BDArmory/Competition/BDACompetitionMode.cs

```diff
@@ -1148,7 +1168,7 @@ void CheckForUncontrolledVessel(Vessel vessel)
         {
             if (vessel == null || vessel.vesselName == null) return;
             if (VesselModuleRegistry.GetModuleCount<MissileFire>(vessel) == 0) return; // The weapon managers are already dead.
-
+            if (vessel.GetName().Contains(BDArmorySettings.PINATA_NAME) && BDArmorySettings.RUNWAY_PROJECT_ROUND == 67) return; //don't delete asteroid pinata
```

Comment:
This should probably apply for all piñatas, not just the S6R7 one.

To avoid potentially weird bug reports, we should probably give a message whenever a piñata craft spawns so that if a regular craft contains the piñata name we can point it out when they complain about weird behaviour.
I think a check after the check for renamed vessels in the `PostSpawnMainSequence` functions would probably be the right place for that.